src/earn/prepareTransactions.ts

```diff
@@ -98,7 +98,7 @@ export async function prepareSupplyTransactions({
     feeCurrencies,
     baseTransactions,
     spendToken: token,
-    spendTokenAmount: new BigNumber(amount),
+    spendTokenAmount: new BigNumber(amount).shiftedBy(token.decimals),
```

Comment:
this is only an issue if the spendToken is also the gas currency. For aave, this can never be the case so there was no user impact